Neos.Flow/Classes/ResourceManagement/ResourceManager.php

```diff
@@ -608,6 +631,44 @@ protected function initializeCollections()
         }
     }
 
+    /**
+     * Decide weather the given media-type has to be sanitized
+     * for now this only checks svg file to solve the issue here https://nvd.nist.gov/vuln/detail/CVE-2023-37611
+     *
+     * @todo create a feature from this and allow to register code for sanitizing file content before importing
+     */
+    protected function isSanitizingRequired(string $mediaType): bool
+    {
+        return $mediaType === 'image/svg+xml';
+    }
+
+    /**
+     * Sanitize file content and remove content that is suspicious
+     * for now this only checks svg file to solve the issue here https://nvd.nist.gov/vuln/detail/CVE-2023-37611
+     *
+     * @todo create a feature from this and allow to register code for sanitizing file content before importing
+     */
+    protected function sanitizeImportedFileContent(string $mediaType, string $content, $filename = ''): string
```

Comment:
would it not make sense to extract this method into an own class the same way what TYPO3 does?

Reply:
Eventually yes. But not in a bugfix. That is why there is "@todo create a feature from this" in there

Reply:
ok